Vagrantfile.fedora

```diff
@@ -29,6 +29,9 @@ EOF
     done
     dnf clean all
 
+    # Configure git
+    git config --global --add safe.directory /vagrant
+
     # Install golang.
     # FIXME go back to golang-go rpm once switched to Fedora 36.
```

Comment:
Maybe this FIXME comment could also be addressed in this pull request?

Reply:
done